<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>salaespera</class>
 <widget class="QWidget" name="salaespera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="cuenta_regresiva">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>en espera</string>
   </property>
  </widget>
  <widget class="QLabel" name="jefe_sala">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Jefe de sala:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="comenzar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Comenzar Ronda</string>
   </property>
  </widget>
  <widget class="QPushButton" name="salir">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
